### PR TITLE
Desktop version.

### PR DESCRIPTION
**Pull request by malembi**
This is the desktop version of my portfolio project. This  desktop version is responsive and runs on any browser without any hiccups.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,7 +221,9 @@ <h2 class="title2">Multi-Post Stories</h2>
   
       <button class="button2" type="button">See project</button>
   </div>
```
Comment (Qoosim) [inline index.html:223]: - [x] It is required to stick to the figma design as much as possible. This includes simulating every layout on the figma design.  Your design must look exactly like that of figma design. Kindly check the `See Project` button on your design as it deos not look like how it is on the figma design. The text should be on a line. You might need to increase the padding of the button to fix ths issue. Ensure you correct this to every other projects

###  Your Design
your-see-project-btn-1

### Figma Design
temp-see-project-btn

- [x] At breakpoint `375px` your mobile version behaves as expected. But from breakpoint `375px` to `768px` white space seems to appear. This is not the expected behaviour of the mobile version. Your app should be responsive. Kindly look into that and fix.

### You app behaviour
[mobile-behaviour.webm](https://user-images.githubusercontent.com/26597432/221517833-7f8de78f-65de-41e9-81f7-86969cb21925.webm)

File: index.html [hunk 2]
```diff
@@ -412,6 +472,7 @@ <h1 class="About">About<br>Myself</h1>
   
     <button type="submit" class="gint" id="gint">Get in touch</button>
   </form>
+</div>
```
Comment (Qoosim) [inline index.html:475]: - [x] Same thing here. Kindly correct this too.

###  Your Design
your-form-btn

###  Figma Design
temp-form-btn

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @malembi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️

- [x] The link to the MIT License in your README file is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set it up properly for your project. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (malembi) [reply index.html:223]: changes required made.

File: index.html (hunk 2 above)
Comment (malembi) [reply index.html:475]: adjusted the buttons as per requested.

**Pull request merged**